lib/logutils/logutils_test.go

```diff
@@ -501,3 +538,35 @@ func (s *mockSyslogWriter) Crit(m string) error {
 	_, err := fmt.Fprintf((*io.PipeWriter)(s), "CRITICAL %s", m)
 	return err
 }
+
+var benchOut bool
+var BenchIn bool
```

Comment:
Not sure what these guys are doing?  Is benchOut just to avoid complaints about the MatchString result being unused?  BenchIn?  Or are they part of the benchmark infrastructure magic?

Reply:
It's a general benchmark-related trick.  Assigning/reading from a global variable prevents the compiler from optimising away the whole loop.